docs/howto/deployment/checklist.txt

```diff
@@ -59,6 +59,17 @@ or from a file::
     with open('/etc/secret_key.txt') as f:
         SECRET_KEY = f.read().strip()
 
+If rotating keys, you may use :setting:`SECRET_KEY_FALLBACKS`::
+
+    import os
+    SECRET_KEY = os.environ['CURRENT_SECRET_KEY']
+    SECRET_KEY_FALLBACKS = [
+        os.environ['OLD_SECRET_KEY'],
+    ]
+
+Ensure that old secret keys are removed from ``SECRET_KEY_FALLBACKS`` in a
+timely manner.
```

Comment:
Should this be `versionadded` since we previously didn't have a way to safely rotate keys?

Reply:
> Should this be `versionadded` since we previously didn't have a way to safely rotate keys?

No, this is an annotation for the section that has changed, but not added. So this section was changed in Django 4.1 in such a way that support for rotating secret keys was added.

Reply:
Thank you for the explanation. I wasn't sure if the added paragraphs counted as a new section, but it sounds like it would need involve a new header or anchored link to justify `versionadded`.